### ISSUE TITLE
User Authentication routes are missing ?

### ISSUE THREAD

**Issue by vibhoo** [solidusio/solidus_starter_frontend#143]
How do I use or access User Authentication can not find it's routes, the user authentication routes (/login) are available in the default solidus gem. Although I have `solidus_auth_devise` installed, and the admin routes are available, is there a way I can enable User Authentication as well ?

**Comment by blocknotes** [comment]
Hey @vibhoo.
Did you replace the `solidus` gem with the single components? (as referred here in the [README](https://github.com/nebulab/solidus_starter_frontend#installation))
If yes, what's the output of `bin/rails routes | grep spree_user` in your project?

**Comment by vibhoo** [comment]
@blocknotes 
Yes I replaced solidus gem with the single components and followed the step 1 to copy the components into my project.
Screen shot of the Gemfile
<img width="269" alt="image" src="https://user-images.githubusercontent.com/488646/104744824-58f15a80-5773-11eb-83d7-07c86f924cec.png">
and when I run `bin/rails routes | grep spree_user` there is no output.

**Comment by blocknotes** [comment]
Can you post your `config/routes.rb` file?

**Comment by vibhoo** [comment]
<img width="950" alt="image" src="https://user-images.githubusercontent.com/488646/104758730-da9db400-5784-11eb-9d24-b9f40cc58a26.png">

**Comment by achalbi** [comment]
Did we find any solution for this?

**Comment by yoerayo** [comment]
I assume this has to do with the route writing in spree_auth_devise being inside this conditional:
`if SolidusSupport.frontend_available? && Spree::Auth::Config.draw_frontend_routes`

**Comment by fthobe** [comment]
@jarednorman low effort issue description and was never brought up again, seems to be solved
